dbschema/scripture.esdl

```diff
@@ -48,4 +48,12 @@ module Scripture {
       constraint expression on (__subject__ >= 0 and __subject__ <= 31101);
     }
   }
+
+  type Portion {
```

Comment:
A portion of scripture could be `Matthew 1-4`. This why I named it `UnspecifiedScripturePortion` in app code.
Maybe `Scripture::UnspecifiedPortion` here?

Reply:
I was wondering what to do there; cool.